packages/now-next/src/index.ts

```diff
@@ -1,55 +1,54 @@
-const { createLambda } = require('@now/build-utils/lambda'); // eslint-disable-line import/no-extraneous-dependencies
-const download = require('@now/build-utils/fs/download'); // eslint-disable-line import/no-extraneous-dependencies
-const FileFsRef = require('@now/build-utils/file-fs-ref'); // eslint-disable-line import/no-extraneous-dependencies
-const FileBlob = require('@now/build-utils/file-blob'); // eslint-disable-line import/no-extraneous-dependencies
-const resolveFrom = require('resolve-from');
-const path = require('path');
-const url = require('url');
-const {
+import { 
+  createLambda,
+  download,
+  FileFsRef,
+  FileBlob,
+  glob,
   runNpmInstall,
   runPackageJsonScript,
-} = require('@now/build-utils/fs/run-user-scripts'); // eslint-disable-line import/no-extraneous-dependencies
-const glob = require('@now/build-utils/fs/glob'); // eslint-disable-line import/no-extraneous-dependencies
-const {
+  Lambda,
+  Files,
+} from '@now/build-utils';
+import resolveFrom from 'resolve-from';
+import path from 'path';
+import url from 'url';
+import {
   readFile,
   writeFile,
-  unlink: unlinkFile,
-  remove: removePath,
+  unlink as unlinkFile,
+  remove as removePath,
   pathExists,
-} = require('fs-extra');
-const semver = require('semver');
-const nextLegacyVersions = require('./legacy-versions');
-const {
+} from 'fs-extra';
+import semver from 'semver';
+import nextLegacyVersions from './legacy-versions';
+import {
   excludeFiles,
   validateEntrypoint,
   includeOnlyEntryDirectory,
   normalizePackageJson,
   onlyStaticDirectory,
   getNextConfig,
-} = require('./utils');
+  getWatchers,
+} from './utils';
 
-/** @typedef { import('@now/build-utils/file-ref').Files } Files */
-/** @typedef { import('@now/build-utils/fs/download').DownloadedFiles } DownloadedFiles */
+interface BuildParamsMeta {
+  isDev: boolean | undefined,
+  requestPath: string | undefined,
+};
 
-/**
- * @typedef {Object} BuildParamsMeta
- * @property {boolean} [isDev] - Files object
- * @property {?string} [requestPath] - Entrypoint specified for the builder
- */
+interface BuildParamsType {
```

Comment:
```suggestion
interface BuildParamsType extends BuildOptions {
```

We already have an exported type for each function.